lib/net.js

```diff
@@ -1647,6 +1647,25 @@ function onconnection(err, clientHandle) {
   }
 
   if (self.maxConnections && self._connections >= self.maxConnections) {
+    if (clientHandle.getsockname || clientHandle.getpeername) {
+      const data = {};
```

Comment:
{} is prone to prototype pollution. Could we use `kEmptyObject` instead of `{}`?

```suggestion
      const data = kEmptyObject;
```

Reply:
I cannot suggest adding an import statement in the GitHub review UI interface. You will need add `const { kEmptyObject } = require('internal/util');` import manually.

Reply:
`kEmptyObject` is a freezed object.

Reply:
Oh, my bad. It should be `Object.create(null)`

Reply:
Done.